crates/rune/src/compile/options.rs

```diff
@@ -61,6 +65,10 @@ impl Options {
             Some("v2") => {
                 self.v2 = it.next() != Some("false");
             }
+            #[cfg(feature = "dynamic_fields")]
```

Comment:
No need to feature gate an option. It might be confusing if it fails to parse despite being in the documentation.

Instead:
* Return a special parse error indicating that the option exists, but it's not supported without enabling a feature; or,
* Emit a warning through `Diagnostics` during a build if the `dynamic_fields` option is used but the feature is disabled.

Reply:
good call.